conda_lock/conda_lock.py

```diff
@@ -1206,6 +1215,11 @@ def lock(
     default=False,
     help="don't attempt to use or install micromamba.",
 )
+@click.option(
+    "--copy",
+    is_flag=True,
+    help="Install using `--copy`.  This is useful for building containers",
```

Comment:
Just because I didn't know offhand what `--copy` does.

```suggestion
    help=(
        "Install using `--copy` to prevent links. "
        "This is useful for building containers"
    ),
```